std/file.d

```diff
@@ -3296,7 +3321,7 @@ private void copyImpl(const(char)[] f, const(char)[] t, const(FSChar)* fromz, co
         //cenforce(core.sys.posix.sys.stat.fstat(fdr, &statbufr) == 0, f, fromz);
 
         immutable fdw = core.sys.posix.fcntl.open(toz,
-                O_CREAT | O_WRONLY, octal!666);
+                O_CREAT | O_WRONLY, std.conv.octal!666);
         cenforce(fdw != -1, t, toz);
```

Comment:
I don't see an `octal` defined anywhere else: can't you just turn the static import above into a selective one?